packages/web-config-server/src/apiV1/dashboard.js

```diff
@@ -92,18 +92,22 @@ export default class extends RouteHandler {
     const views = [];
     for (let i = 0; i < dashboardReportIds.length; i++) {
       const viewId = dashboardReportIds[i];
-      const report = await DashboardReport.findOne({
+      const reports = await DashboardReport.find({
         id: viewId,
-        drillDownLevel: null, //drillDownLevel = null so that only the parent reports are selected, we don't want drill down reports at this level.
       });
 
-      const { viewJson, dataBuilder } = report;
-      const { displayOnEntityConditions, ...restOfViewJson } = viewJson; //Avoid sending displayOnEntityConditions to the frontend
-      const view = { viewId, ...restOfViewJson, requiresDataFetch: !!dataBuilder };
+      reports.forEach(report => {
+        const { viewJson, dataBuilder, drillDownLevel } = report;
+        const { displayOnEntityConditions, ...restOfViewJson } = viewJson; //Avoid sending displayOnEntityConditions to the frontend
+        const viewKey = drillDownLevel ? `${viewId}_${drillDownLevel}` : viewId;
```

Comment:
This is a good strategy, but the fact that a drill down report gets a composite key should only be handled by _either_ the front end or the back end. Can we either:
- send through the plain `viewId` from this endpoint, and get the frontend to do the key building in both places required; or
- do the key building here and also on the dashboard report data fetch, or wherever else the back end might be sending through (or interpreting as a query parameter) a viewId?